scripts/install/install-pocs.sh

```diff
@@ -50,15 +50,15 @@ usage() {
  If in DEVELOPER mode, the following options are also available:
   USER      The PANUSER environment variable, defaults to current user (i.e. PANUSER=$USER).
   PANDIR    Default install directory, defaults to PANDIR=${PANDIR}. Saved as PANDIR
-            environment variable.
+            environment variable. Test $LOGFILE.
 "
 }
 
 
 DEVELOPER=${DEVELOPER:-false}
 PANUSER=${PANUSER:-$USER}
 PANDIR=${PANDIR:-/var/panoptes}
-LOGFILE="${PANDIR}/install-pocs.log"
+LOGFILE=${PANDIR}/install-pocs.log
```

Comment:
Was this necessary to make it work? Just curious.

Reply:
No, it actually doesn't seem to make a difference.